src/plugins/explore/common/index.ts

```diff
@@ -16,6 +16,7 @@ export const CONTEXT_DEFAULT_SIZE_SETTING = 'context:defaultSize';
 export const CONTEXT_STEP_SETTING = 'context:step';
 export const CONTEXT_TIE_BREAKER_FIELDS_SETTING = 'context:tieBreakerFields';
 export const MODIFY_COLUMNS_ON_SWITCH = 'discover:modifyColumnsOnSwitch';
+export const DEFAULT_TRACE_COLUMNS_SETTING = 'explore:defaultTraceColumns';
```

Comment:
imo we should stick to the naming pattern like defaultColumns:traces or traces:defaultColumns. this keeps settings grouped together since defaultColumns controls logs. using same pattern makes settings easier to find